test/Microsoft.TestPlatform.PerformanceTests/ProtocolV1Tests.cs

```diff
@@ -30,97 +32,96 @@ public class ProtocolV1Tests
         Traits = { new Trait("Priority", "0"), new Trait("Category", "unit") }
     };
 
-    private static DateTimeOffset s_startTime = new(new DateTime(2007, 3, 10, 0, 0, 0, DateTimeKind.Utc));
+    private static readonly DateTimeOffset StartTime = new(new DateTime(2007, 3, 10, 0, 0, 0, DateTimeKind.Utc));
 
     private static readonly TestResult TestResult = new(TestCase)
     {
         // Attachments = ?
         // Messages = ?
+        // Different text values because they might be cached?
         Outcome = TestOutcome.Passed,
         ErrorMessage = "sampleError",
         ErrorStackTrace = "sampleStackTrace",
         DisplayName = "sampleTestResult",
         ComputerName = "sampleComputerName",
         Duration = TimeSpan.MaxValue,
-        StartTime = s_startTime,
+        StartTime = StartTime,
         EndTime = DateTimeOffset.MaxValue
     };
 
     [TestMethod]
     public void TestCaseSerialize()
     {
-        Serialize(TestCase);
+        SerializeV1(TestCase);
         Stopwatch sw = Stopwatch.StartNew();
 
         for (int i = 0; i < 10000; i++)
         {
-            Serialize(TestCase);
+            SerializeV1(TestCase);
         }
         sw.Stop();
 
-        VerifyPerformanceResult("TestCaseSerialize1", 2000, sw.ElapsedMilliseconds);
+        var actualDuration = sw.Elapsed;
+        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), $"when serializing 10k test cases");
```

Comment:
Nit: interpolation is not needed.

```suggestion
        actualDuration.Should().BeLessOrEqualTo(2.Seconds(), "when serializing 10k test cases");
```